Plugins/DeleteUnusedBranches/DataGridViewCheckBoxHeaderCell.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Drawing;
+using System.Windows.Forms;
+using System.Windows.Forms.VisualStyles;
+
+namespace DeleteUnusedBranches
+{
+    public delegate void CheckBoxClickedHandler(bool state);
+
+    public class DataGridViewCheckBoxHeaderCellEventArgs : EventArgs
```

Comment:
one class per file

Reply:
Done.  Also renamed it to `CheckBoxHeaderCellEventArgs`